src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/Transmission/OtlpExporterTransmissionHandler.cs

```diff
@@ -121,7 +121,7 @@ protected bool TryRetryRequest(TRequest request, DateTime deadlineUtc, out Expor
         response = this.ExportClient.SendExportRequest(request, deadlineUtc);
         if (!response.Success)
         {
-            OpenTelemetryProtocolExporterEventSource.Log.ExportMethodException(response.Exception, isRetry: true);
+            OpenTelemetryProtocolExporterEventSource.Log.ExportMethodException(response.Exception!, isRetry: true);
```

Comment:
Is this an assumption that when Success is false, Exception is not null?

Reply:
This is an existing implementation and will be removed in next few follow up PRs. In the current implementation exception is optional.